.github/workflows/ci.yml

```diff
@@ -21,14 +21,23 @@ jobs:
           - stable
     steps:
       - uses: actions/checkout@v3
+        with:
+          submodules: recursive
       - run: rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }}
       - run: rustup target add wasm32-unknown-unknown
-      - name: Build required wasm
+      - run: cargo install --locked --version 20.3.0 soroban-cli --features opt
+      - name: Build required did contract wasm
+        run: |
+          soroban contract build --package soroban-did-contract
+      - name: Optimize soroban_did_contract.wasm
+        run: |
+          soroban contract optimize --wasm target/wasm32-unknown-unknown/release/soroban_did_contract.wasm
+      - name: Build required vault contract wasm
         run: |
-          cargo build --target wasm32-unknown-unknown --release --package vault-contract
+          soroban contract build --package vault-contract
       - name: Contract Build
```

Comment:
If we remove this step, the CI will continue working or not? In my local it works.